Orange/widgets/regression/owunivariateregression.py

```diff
@@ -109,6 +111,16 @@ def __init__(self):
 
         self.apply()
 
+    def send_report(self):
+        if(self.data == None):
```

Comment:
Remove parentheses.

Use `self.data is None` instead of `self.data == None`. (PEP 8: *Comparisons to singletons like None should always be done with is or is not, never the equality operators.*)